config/logstash.yml

```diff
@@ -50,6 +50,16 @@
 #
 # pipeline.unsafe_shutdown: false
 #
+# Defaults to false. Setting this to true will rescue all exceptions in the
+# pipeline.This means that if there are any errors in filters, or if a
+# conditional raises an exception logstash will drop the event, log an
+# error-level message, and continue processing.
+# If set to false logstash will immediately exit with an error in these
```

Comment:
I'm going to add in "If set to false logstash will immediately exit with an error in these
# conditions, which may be more safe since potentially fewer events would be dropped."